test/core/Bounds.js

```diff
@@ -88,17 +83,15 @@ describe('getBounds', function ()
         expect(bounds.height).to.equal(40);
     });
 
-    it('should register correct width/height with an empty Container', function ()
+    it('should register correct width/height with an empty Container', () =>
     {
-        var parent = new PIXI.Container();
+        const parent = new PIXI.Container();
 
-        var container = new PIXI.Container();// Graphics().beginFill(0xFF0000).drawCircle(0, 0, 10, 10);//texture);
+        const container = new PIXI.Container();// Graphics().beginFill(0xFF0000).drawCircle(0, 0, 10, 10);//texture);
```

Comment:
Are the inline comments here really needed? Can we just remove them (as well as all the subsequence copy/pasted version of this)?